### DISCUSSION TITLE
Printing all Project Tasks related to a Project in a PDF

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#16896] [How to] [answered]
Hi all, similar to my previous question of printing all products within a quote in a PDF, I was wondering how I would print all Project Tasks related to a specific Project in a PDF. My current PDF format is that I have all the project info first, and then below that I want a table that lists each project's name, who it is assigned to, when it is due, and its description.

Thank you all in advance!

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/blob/1116b3b9cdd0ff68f5b1addb5fd134a007d80801/app/TextParser.php#L41

**Reply by mariuszkrzaczkowski** [reply]
`$(relatedRecordsList : ProjectTask)$`

**Reply by bfeddes** [reply]
How would I make this appear as a table similar to how products are displayed after printing out a Quote? Is there also a way so that the description of the task is displayed below the rest of the information? 
Example: 
Task Name: Assigned To: Status
Description:
